### PR TITLE
Main: Report triggers with unknown arguments.

### PR DESCRIPTION
**Pull request by Gacko**
Towards https://github.com/giantswarm/giantswarm/issues/31991.

See this test PR for look and feel: https://github.com/giantswarm/tinkerers-ci-test-1/pull/29#issuecomment-2460602582

### CODE REVIEW

**Comment by Gacko** [comment]
See this test PR for look and feel: https://github.com/giantswarm/tinkerers-ci-test-1/pull/29#issuecomment-2460602582

File: main.go [hunk 1]
```diff
@@ -412,11 +457,12 @@ func getPipeline(ctx context.Context, pipelineName string, userProvidedNamespace
 			continue
 		} else if err == nil {
 			return pipeline, namespace, nil
-		} else if err != nil {
```
Comment (AverageMarcus) [inline main.go:415]: 🤨  Why this change? Do we not still need it?

Comment (Gacko) [reply main.go:415]: `err` is definitely `nil` when you reach this line and either static checks or `gopls` were hinting that. 🙂

Comment (AverageMarcus) [reply main.go:415]: 🤦  Oh yeah...

**Comment by Gacko** [comment]
Final look and feel: https://github.com/giantswarm/tinkerers-ci-test-1/pull/29#issuecomment-2465497594

I decided to create the comments per trigger. Otherwise I would have needed to come up with a complex logic for storing triggers with their unknown arguments just to concatenate them in a single comment later.

Most the time users are running a single trigger per comment only and the resulting pipeline runs then also create single comments, so I don't think it's worth the effort.
